packages/login/src/login.stories.js

```diff
@@ -1,37 +1,57 @@
 import React from 'react'
 import {storiesOf} from '@storybook/react'
-import {withKnobs, boolean, select, text} from '@storybook/addon-knobs'
+import {withKnobs, boolean, text} from '@storybook/addon-knobs'
 import {action} from '@storybook/addon-actions'
+import {linkTo} from '@storybook/addon-links'
+import {injectIntl, intlShape} from 'react-intl'
 
 import LoginApp from './main'
+import Readme from '../README.md'
 
-storiesOf('Apps/Login', module)
+storiesOf('Apps|Login', module)
   .addDecorator(withKnobs)
   .add(
     'Login',
-    () => <LoginAppWrapper/>
+    () => <LoginAppWrapperStoryIntl/>,
+    {info: {disable: true}, notes: Readme}
   )
 
-class LoginAppWrapper extends React.Component {
+class LoginAppWrapperStory extends React.Component {
   constructor(props) {
     super(props)
     this.childKey = 0
   }
 
+  confirmed = false
```

Comment:
why is that done this way? and what about `childKey`? looks a bit hacky to me.

Reply:
Sure looks hacky. that's because it is ;)
First of all we talking about a story. So none productive code. I think hacky is ok in this case.

We need to ensure that the component is entirely rerendered on knob (input) changes and changing the key prop a easy solution for this problem. Also see:
https://medium.com/@albertogasparin/forcing-state-reset-on-a-react-component-by-using-the-key-prop-14b36cd7448e 

I think the whole discussion about how to properly change input params and reload the app needs to be done as soon as we ship our apps to be used independently (CMS widgets and so on).